enterprise/micronaut/nbproject/project.xml

```diff
@@ -68,6 +68,32 @@
                         <specification-version>1.18</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.maven</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <release-version>2</release-version>
+                        <specification-version>2.154</specification-version>
+                    </run-dependency>
+                </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.project.dependency</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <implementation-version/>
+                    </run-dependency>
+                </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.maven.embedder</code-name-base>
```

Comment:
These modules don't provide real API. Depending on them from foreign "micronaut" module is strange. Also adding a dependency on Maven support, when "micronaut" supports both, Maven and Gradle is strange.

Reply:
re. depending from Micronaut no Gradle | Maven - agreed, a solution would be to create eager micronaut-maven + micronaut-gradle modules. 

re. no real API - hm, I need to use the maven model types, i.e. `org.apache.maven.model.PluginExecution`. These are currently exposed to the module system by `maven.emebdder`. Is there a better way how to import those types ?